### ISSUE TITLE
read_cdf should flag likely problems

### ISSUE THREAD

**Issue by almartin82** [almartin82/mapvizieR#246]
for instance, if students by school has dupes on studentid in the same term (ie, two different kids with studentid 0000) mapvizieR will fail.

perhaps write a wrapper that is aggressive about discarding these records (and warning you that they were discarded, so you can follow up asynchronously)?

**Comment by chrishaid** [comment]
I agree with the sentiment here!  Especially the aggressive with a warning.  The question I have is should we do it with `read_cdf()`, which **some** sers will use, or as part of the error checking in `mapvizieR.defualt`, which **all** users will use?  I'd think the latter would be safer.
